app/Http/Controllers/API/UserController.php

```diff
@@ -20,4 +22,22 @@ public function register(RegisterRequest $request)
 
         return AuthenticationResource::make($token);
     }
+
+    public function login(LoginRequest $request)
```

Comment:
watch laravel document before start coidng like this.

https://laravel.com/docs/9.x/authentication#authenticating-users